redaxo/src/addons/media_manager/tests/media_manager_test.php

```diff
@@ -48,8 +48,8 @@ public function testCreate()
 
             $manager = rex_media_manager::create('non_existing_type', $filename);
 
-            static::assertFileNotExists($manager->getCacheFilename());
-            static::assertFileNotExists($manager->getHeaderCacheFilename());
+            static::assertFileDoesNotExist($manager->getCacheFilename());
```

Comment:
hmm.. es scheint wir nutzen diese deprecated methoden, damit wir alte php versionen unterstützen können..?

Reply:
So ist es. In psalm ignoriere ich die Methoden deshalb: https://github.com/redaxo/redaxo/blob/0183c534d1ef8054ae3428b4cd6576f38f19d98d/psalm.xml#L59-L65

Alternativ könnten wir hier mit Versionsweichen arbeiten.

Reply:
Die Problematik mit den phpunit-methoden hat sich nun erledigt und wir können problemlos die neuen methoden verwenden.
(Die Zeilen oben können dann entsprechend in der psalm.xml entfernt werden)

Reply:
somit sind wir hier auch gree <3